The slack notification is no more received when the "Propagate documentation " workflow failed

1st failing workflow without slack notification: 29th march

![image](https://user-images.githubusercontent.com/27200110/237054482-7ea7b9fe-732e-45bf-959c-17272385e077.png)



Logs on 2023-05-08 of https://github.com/bonitasoft/bonita-documentation-site/actions/runs/4910942473/jobs/8768540157

<details>
<summary>Details</summary>
<pre>
0s
Run cd ./bonita-doc
########################################
Performing the doc propagation
########################################
> Propagating documentation upwards
> Configuration: NO_PUSH=false
> Configuration: REPO_NAME=bonita-doc
> Merging branch '2021.2' into '2022.1'
Switched to a new branch '2022.1'
branch '2022.1' set up to track 'origin/2022.1'.
Already up to date.
Auto-merging modules/ROOT/pages/bonita-data-repository-dependencies.adoc
Auto-merging modules/ROOT/pages/bonita-engine-dependencies.adoc
Auto-merging modules/ROOT/pages/bonita-ui-designer-dependencies.adoc
Auto-merging modules/ROOT/pages/bonita-web-dependencies.adoc
Auto-merging modules/ROOT/pages/portal-js-dependencies.adoc
Auto-merging modules/identity/pages/ldap-synchronizer.adoc
CONFLICT (content): Merge conflict in modules/identity/pages/ldap-synchronizer.adoc
Automatic merge failed; fix conflicts and then commit the result.
Error: Process completed with exit code 1.

Run bonitasoft/actions/packages/notify-slack@v2
  with:
    CHANNEL_ID: CCQGMR4ES
    MESSAGE: :fire:  Propagating documentation upwards failed for repository *bonita-doc*. \n \n  
  @channel *We need someone* ! \n  
  - Add a :fire_extinguisher:if you take the action to resolve the conflicts (only one person is required) \n 
  - Add a :sweat_drops: when it’s done (and eventually a :party_parrot: ) \n \n
  More details on solving the conflicts locally <https://bonitasoft.atlassian.net/wiki/spaces/BS/pages/1181483014/Bonita+documentation+site+content+update#Bonita| there>
  
  env:
    SLACK_BOT_TOKEN: ***
Run slackapi/slack-github-action@v1.23.0
  with:
    channel-id: CCQGMR4ES
    payload: {
      "blocks": [
          {
              "type": "section",
              "text": {
                  "type": "mrkdwn",
                  "text": ":fire:  Propagating documentation upwards failed for repository *bonita-doc*. \\n \\n  \n@channel *We need someone* ! \\n  \n- Add a :fire_extinguisher:if you take the action to resolve the conflicts (only one person is required) \\n \n- Add a :sweat_drops: when it’s done (and eventually a :party_parrot: ) \\n \\n\nMore details on solving the conflicts locally <https://bonitasoft.atlassian.net/wiki/spaces/BS/pages/1181483014/Bonita+documentation+site+content+update#Bonita| there>\n"
              }
          },
          {
              "type": "divider"
          },
          {
              "type": "section",
              "text": {
                  "type": "mrkdwn",
                  "text": "More details <https://github.com/bonitasoft/bonita-documentation-site/actions/runs/4910942473?check_suite_focus=true| here>"
              }
          }
      ]
  }
  
  env:
    SLACK_BOT_TOKEN
</pre>
</details>

--- a/.github/workflows/propagate-doc-upwards.yml
+++ b/.github/workflows/propagate-doc-upwards.yml
@@ -57,10 +57,9 @@ jobs:
         with:
           CHANNEL_ID: ${{ vars.SLACK_CHANNEL_ID }}
           MESSAGE: |
-            :fire:  Propagating documentation upwards failed for repository *${{matrix.repo}}*. \n \n  
-            @channel *We need someone* ! \n  
-            - Add a :fire_extinguisher:if you take the action to resolve the conflicts (only one person is required) \n 
-            - Add a :sweat_drops: when it’s done (and eventually a :party_parrot: ) \n \n
+            :fire:  Propagating documentation upwards failed for repository *${{matrix.repo}}*.
+            @channel *We need someone*!
+            - Add a :fire_extinguisher:if you take the action to resolve the conflicts (only one person is required)
+            - Add a :sweat_drops: when it’s done (and eventually a :party_parrot:)
             More details on solving the conflicts locally <https://bonitasoft.atlassian.net/wiki/spaces/BS/pages/1181483014/Bonita+documentation+site+content+update#Bonita| there>
-        env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}

